.projenrc.ts

```diff
@@ -38,7 +39,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   projenVersion: '~0.78.8',
   constructsVersion: '10.3.0',
   defaultReleaseBranch: 'main',
-  jsiiVersion: '~5.1.0',
+  jsiiVersion: '~5.3.0',
```

Comment:
Can we remove this change from this PR ? not related to the other features and would require further testing